### PR TITLE
Enabled integration tests on CoreCLR

### LINKED ISSUE #1164: Run integrations tests on CoreCLR too

### PR DESCRIPTION
**Pull request by devlead**
* fixes #1164

### CODE REVIEW

File: tests/integration/build.sh [hunk 1]
```diff
@@ -93,6 +122,12 @@ if [ ! -f $CAKE_EXE ]; then
     exit 1
 fi
 
+# Ensure that Cake CoreLCR can be found where we expect it to.
+if [ ! -f $CAKE_EXE ]; then
```
Comment (patriksvensson) [inline tests/integration/build.sh]: Shouldn't this be `$CAKE_DLL`?

Comment (devlead) [inline tests/integration/build.sh]: Yes it should, I'll fix.

Comment (devlead) [inline tests/integration/build.sh]: @patriksvensson adjusted.

**Comment by patriksvensson** [comment]
Merged!

**Pull request merged**